force_bdss/mco/optimizer_engines/i_optimizer.py

```diff
@@ -0,0 +1,15 @@
+import abc
```

Comment:
Let's import this  class in the `force_bdss.api` module so it can be used by other plugins